glide-core/src/client/value_conversion.rs

```diff
@@ -1010,7 +1010,7 @@ fn convert_inner_map_by_type(
     key_type: Option<ExpectedReturnType>,
     value_type: Option<ExpectedReturnType>,
 ) -> RedisResult<Value> {
-    let result = map
+    let result: Result<Vec<(Value, Value)>, redis::RedisError> = map
```

Comment:
Do we need this?

Reply:
If this is explicitly annotated, then the turbofish at line 1021 could probably be removed entirely.